Formula/arangodb.rb

```diff
@@ -6,6 +6,7 @@ class Arangodb < Formula
   head "https://github.com/arangodb/arangodb.git", :branch => "unstable"
 
   bottle do
+    revision 1
```

Comment:
The `revision 1` line should go outside the `bottle do` block. It needs to apply to the entire formula, not just the bottle build.